services/masterdata/src/lib/core/interfaces/query-model.interface.ts

```diff
@@ -0,0 +1,14 @@
+export interface IQueryModel {
+  offset?: number | null | undefined;
+  limit?: number | null | undefined;
+  noLimit?: boolean | null | undefined;
+  tag?: string | null | undefined; // @deprecated Please use the tags instead
```

Comment:
Doublecheck: do we have the deprecated tag also in the backend (on the QueryModel class)?

Reply:
Remove if no longer necessary.

Reply:
@Johan: yes (marked with obsolete)